styles.css

```diff
@@ -62,9 +62,291 @@ body {
-  margin: 8px;
+  align-items: center;
+  margin: 6px;
+}
+
+.social-media-icons img {
+  padding: 6px;
+}
+
+.works {
+  display: grid;
+  flex-direction: column;
+  align-items: center;
+  height: 2550px;
+  padding: 114px 24px;
+}
+
+.card-works {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 16px;
+  width: 327px;
+  height: 630px;
+  left: calc(50% - 327px / 2);
+  top: 100px;
+  background: #fff;
+  border: 1px solid #dfe1e6;
+  box-sizing: border-box;
+  border-radius: 16px;
+  margin: 30px 0;
+}
+
+.snapshoot-portfolio {
+  height: 220px;
+  margin: 12px 0;
+}
+
+.portfolio-snapshoot {
+  border-radius: 8px;
+}
+
+.left-block {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  padding: 0;
+  width: 295px;
+  height: 288px;
+  left: 16px;
+  top: 248px;
+  margin: 12px 0;
+}
+
+.project-title {
+  font-weight: 700;
+  font-size: 24px;
+  color: #344563;
+}
+
+.summary-frame {
+  display: flex;
+  flex-direction: row;
+  padding: 0;
+}
+
+.client {
+  margin-right: 10px;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  color: #344563;
+}
+
+.role {
+  margin-right: 10px;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  color: #7a869a;
+}
+
+.year {
+  margin-right: 10px;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  color: #7a869a;
+}
+
+.project-description p {
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #344563;
+}
+
+.tags {
+  display: flex;
+  flex-direction: row;
+}
+
+.project-button {
+  position: relative;
+  display: inline-block;
+  padding: 12px;
+  width: 130px;
+  height: 48px;
+  border: 1px solid #6070ff;
+  box-sizing: border-box;
+  border-radius: 8px;
+  background: #fff;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 14px;
+  align-items: center;
+  line-height: 24px;
+  color: #396df2;
+  margin: 0 10px;
+  cursor: pointer;
+  right: 10px;
+}
+
+.project-button:hover {
+  background: #6070ff;
+  color: #fff;
+}
+
+.about-myself {
+  background: #fff;
+  display: grid;
+  align-items: center;
+  padding: 114px 24px;
+}
+
+.username {
+  color: #172b4d;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  display: flex;
+  align-items: center;
+}
+
+.get-resume-button {
+  display: inline-block;
+  width: 150px;
+  height: 48px;
+  border: 1px solid #6070ff;
+  box-sizing: border-box;
+  border-radius: 8px;
+  background: #fff;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 14px;
+  align-items: center;
+  line-height: 24px;
+  color: #396df2;
+  cursor: pointer;
+  margin: 12px 8px;
+}
+
+.get-resume-button:hover {
+  background: #6070ff;
+  color: #fff;
+}
+
+.skill-list {
+  position: relative;
+  display: flex;
+  flex-direction: column;
+  padding: 16px;
+  margin: 12px 8px;
+  color: #344563;
+  right: 70px;
+}
```

Comment:
Please kindly, update the css `margin` of your skill list to stick to Figma template.
As you see in your design the left margin is not equal to the right margin.

Your design:

<img width="444" alt="image" src="https://user-images.githubusercontent.com/7243132/170823317-8512299f-c1c4-49b9-80a8-d0b1bc1fca95.png">

Figma:
<img width="377" alt="image" src="https://user-images.githubusercontent.com/7243132/170823329-5f380c06-9a7b-42aa-af7f-235bd008a80d.png">